commands/command_fetch.go

```diff
@@ -21,8 +22,65 @@ var (
 	fetchAllArg    bool
 	fetchPruneArg  bool
 	fetchDryRunArg bool
+	fetchJsonArg   bool
 )
 
+type FetchWatcher struct {
```

Comment:
As we don't need to export this structure outside of the `commands` package, we can just call it `fetchWatcher`.